flocker/control/httpapi.py

```diff
@@ -836,6 +837,45 @@ def list_current_nodes(self):
                 for node in
                 self.cluster_state_service.as_deployment().nodes]
 
+    @app.route("/state/nodes/by_era/<era>", methods=['GET'])
+    @user_documentation(
+        u"""
+        If you are talking to the API from a process on a node then this
```

Comment:
Do you want to actually define era explicitly? I think can figure it out from context, but if I was just looking at the docs as an external user this would be at best just noise that I'd have to filter out.